tests/test_http_competition_download.py

```diff
@@ -0,0 +1,169 @@
+import os
+from datetime import datetime, timezone
+from typing import Optional
+
+import requests
+
+import kagglehub
+from kagglehub.cache import COMPETITIONS_CACHE_SUBFOLDER, get_cached_archive_path
+from kagglehub.handle import parse_competition_handle
+from tests.fixtures import BaseTestCase
+
+from .server_stubs import competition_download_stub as stub
+from .server_stubs import serv
+from .utils import create_test_cache
+
+INVALID_ARCHIVE_COMPETITION_HANDLE = "invalid/invalid"
+COMPETITION_HANDLE = "titanic"
+COMPETITION_2_HANDLE = "titanic"
```

Comment:
Is this used anywhere else? If not, good to remove

Reply:
COMPETITION_2_HANDLE not used.